src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  font-family: Robotos, sans-serif;
+  font-style: normal;
+
+  &__picture {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
```

Comment:
```suggestion

```
Avoid hardcoding sizes of the blocks, it should depend on the content. If you want to have indents, use padding for the block or parent block instead